ietf/doc/utils.py

```diff
@@ -1059,7 +1059,7 @@ def augment_docs_and_user_with_user_info(docs, user):
                 reviewteamsettings__isnull=False, role__person__user=user, role__name='reviewer')
 
         doc_pks = [d.pk for d in docs]
-        clist = CommunityList.objects.filter(user=user).first()
+        clist = CommunityList.objects.filter(person=Person.objects.get(user=user)).first()
```

Comment:
Another place to move `user` out of the `Community` app. I think this method is only used in a couple places, so it'd probably be better to move the authentication check and `Person` lookup out of this method and into the caller. The method name and signature both need a change.

Reply:
Technically, this is outside of the `Community` app, but it wasn't a huge push.